templates/THT/myservice-container-ansible.yaml.j2

```diff
@@ -66,42 +36,59 @@ parameters:
     type: json
 
 resources:
-  ContainersCommon:
-    type: ../containers-common.yaml
+  #############################################################################
+  # TODO: Check that indentation is correct (list items don't indent)
+  #############################################################################
+  {{ tht.resources | to_nice_yaml(width=80, indent=2) | indent(width=2, indentfirst=false)}}
 
   RoleParametersValue:
     type: OS::Heat::Value
     properties:
       type: json
       value:
         map_merge:
-          - tripleo_role_name: {get_param: RoleName}
           #####################################################################
-          # The first items in this list should be any existing
-          # {{ myservice}}-related parameters, using a new key that will be
-          # available in the tripleo_{{myservice }} ansible role.
+          # The values here will be available in the tripleo_{{ myservice }}
+          # (and the {{ myservice }} ansible role.
+          #####################################################################
+          # TODO: Update names as appropriate, consider using tripleo_* as a
```

Comment:
I no longer recommend updating anything here. It's unintuitive for the operator using the new THT templates.

In the tripleo_myservice ansible role, these can be re-mapped, if appropriate.